src/Rules/NoEnvCallsOutsideOfConfigRule.php

```diff
@@ -73,6 +73,7 @@ public function processNode(Node $node, Scope $scope): array
     protected function isCalledOutsideOfConfig(FuncCall $call, Scope $scope): bool
     {
         foreach ($this->configDirectories as $configDirectoryGlob) {
+            $configDirectoryGlob = str_replace('/', DIRECTORY_SEPARATOR, $configDirectoryGlob);
```

Comment:
I think we can do this in constructor. And even use `$this->fileHelper->normalizePath(...)`. It covers some other edge cases too. What do you think?